evalml/pipelines/components/transformers/encoders/ordinal_encoder.py

```diff
@@ -49,6 +49,7 @@ class OrdinalEncoder(Transformer, metaclass=OrdinalEncoderMeta):
     name = "Ordinal Encoder"
     hyperparameter_ranges = {}
     """{}"""
+    # --> should I add a fit_transform method?
```

Comment:
Only if you think the standard `Transformer.fit_transform()` doesn't cover it!

Reply:
I don't see a reason to add `fit_transform` (seems like it's mostly used for testing?), so I haven't added one